components/dashboard/src/app/AppRoutes.tsx

```diff
@@ -80,10 +80,12 @@ const ConfigurationDetailPage = React.lazy(
     () => import(/* webpackPrefetch: true */ "../repositories/detail/ConfigurationDetailPage"),
 );
 
+const PrebuildListPage = React.lazy(() => import(/* webpackPrefetch: true */ "../prebuilds/list/PrebuildList"));
+
 export const AppRoutes = () => {
     const hash = getURLHash();
     const location = useLocation();
-    const repoConfigListAndDetail = useFeatureFlag("repoConfigListAndDetail");
+    const configurationsAndPrebuilds = useFeatureFlag("configurationsAndPrebuilds");
```

Comment:
Add a comment to be resolved in case we forget it, there're several flags can affect APIs